kickstarts/base.ks.erb

```diff
@@ -3,6 +3,7 @@
 logging --level=debug
 
 #version=DEVEL
+text
```

Comment:
Does this reduce the size or was this for debugging...I thought we already had this somewhere, which is part of the reason I was asking.  @bdunne?